linkerd/yaml.go

```diff
@@ -68,7 +68,7 @@ func (d *YAMLDecoder) Close() error {
 }
 
 const yamlSeparator = "\n---"
-const separator = "---"
```

Comment:
@gunjanmimo can you explain what exactly is this change is doing and why it is required?

Reply:
this "_" I used to avoid the error message "declared bu not used"

Reply:
Okay @gunjanmimo Thanks, Got it.

Reply:
@nupur-thakur do you know, why git action is not showing me all the checks?